slangpy/tests/slangpy_tests/test_load_image.py

```
@@ -0,0 +1,145 @@
+def test_load_greyscale_single_channel(device, grey_image):
+    """Load a single-channel image."""
+    path, expected = grey_image
+    tensor = spy.Tensor.load_from_image(device, path)
+    result = tensor.to_numpy()
+    assert result.shape == expected.shape
+    assert np.allclose(result, expected, atol=1e-5)
+
+
+def test_load_two_channel(device, rg_image):
+    """Load a 2-channel image and verify it produces float2 tensor."""
+    path, expected = rg_image
```

Ruff (0.15.9)

[warning] Unpacked variable `expected` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)